src/pytorch_lightning/utilities/data.py

```diff
@@ -350,17 +382,36 @@ def _dataloader_init_kwargs_resolve_sampler(
     return {"sampler": sampler, "shuffle": False, "batch_sampler": None}
 
 
+def _inject_arg_to_saved(
+    args: List[Any], kwargs: Dict[str, Any], arg_names: List[str], inject_name: str, inject_object: Any
```

Comment:
super nit: I would put the inject argument key and value first in the signature

Reply:
I'm not sure I agree, but I don't have a strong preference either way, so changed according to your suggestion :smile: